src/main/java/donnafin/logic/PersonAdapter.java

```diff
@@ -150,67 +169,116 @@ private Person editPersonTags(Person personToEdit, String newValue) {
         Set<Tag> newTags = Arrays.stream(tagStrings)
                 .map(Tag::new)
                 .collect(Collectors.toSet());
+
         return new Person(
-                personToEdit.getName(), personToEdit.getPhone(), personToEdit.getEmail(),
-                personToEdit.getAddress(), newTags, personToEdit.getNotes(),
+                personToEdit.getName(),
+                personToEdit.getPhone(),
+                personToEdit.getEmail(),
+                personToEdit.getAddress(),
+                newTags,
+                personToEdit.getNotes(),
                 personToEdit.getPolicies(),
-                personToEdit.getLiability(), personToEdit.getCommission(),
-                personToEdit.getAssetSet());
+                personToEdit.getLiabilities(),
+                personToEdit.getAssets());
     }
 
     private Person editPersonNotes(Person personToEdit, String newValue) {
         Notes newNotes = new Notes(newValue);
+
         return new Person(
-                personToEdit.getName(), personToEdit.getPhone(), personToEdit.getEmail(),
-                personToEdit.getAddress(), personToEdit.getTags(), newNotes,
+                personToEdit.getName(),
+                personToEdit.getPhone(),
+                personToEdit.getEmail(),
+                personToEdit.getAddress(),
+                personToEdit.getTags(),
+                newNotes,
                 personToEdit.getPolicies(),
-                personToEdit.getLiability(), personToEdit.getCommission(),
-                personToEdit.getAssetSet());
+                personToEdit.getLiabilities(),
+                personToEdit.getAssets());
+    }
+
+    private Policy stringToPolicy(String policyString) {
+        String[] details = policyString.split(Attribute.DELIMITER);
+        //@parserteam do error handling for this
+        //        if (details != 5) {
+        //            throw new ParseException(Policy.MESSAGE_CONSTRAINTS);
+        //        }
+
+        return new Policy(details[0], details[1], details[2], details[3], details[4]);
```

Comment:
I believe this whole thing should be handled by the parser util stuff you wrote?